src/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
no need to set padding 0 for body, it doesn't have default paddings

Reply:
ok, thnx!